src/resolver/Resolver.ts

```diff
@@ -78,3 +76,25 @@ export class Resolver {
     });
   }
 }
+
+/**
+ * Packaging the resolver as a driver to meet https://github.com/decentralized-identity/did-resolver spec
+ */
+export function getResolver() {
+  const resolver: Resolver = new Resolver();
+
+  async function resolve(
+    did: string,
+    _parsed: ParsedDID,
+    _didResolver: DrResolver,
+    _options: DIDResolutionOptions,
+    resolveUrl: string
```

Comment:
I don't get this. From an integration point one expects to use getResolve which returns the resolve function right?

The resolve function is the contract all the resolvers use. So I would expect getResolver to have our custom options, and configure the resolver that is returned by the standard resolver function

Reply:
sorry about that I didn't get the point you were making yesterday.
 now I'm using it like this: `options['resolveUrl']` (and the `resolveUrl` is removed of course